### DISCUSSION TITLE
What breaking change motivated v5.0.0?

### DISCUSSION THREAD

**Discussion by rootulp** [golangci/golangci-lint-action#1025] [Q&A] [answered]
I want to upgrade from v4.0.0 to [v5.0.0](https://github.com/golangci/golangci-lint-action/releases/tag/v5.0.0) but I don't know what breaking changes motivated the release. Can the release notes be amended to describe any breaking changes?

**Comment by ldez** [comment, marked as answer]
https://github.com/golangci/golangci-lint-action?tab=readme-ov-file#compatibility

https://github.com/golangci/golangci-lint-action/releases/tag/v5.0.0

**Reply by rootulp** [reply]
Thanks for adding that to https://github.com/golangci/golangci-lint-action/releases/tag/v5.0.0 👍
